index.html

```diff
@@ -0,0 +1,250 @@
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="single-tech">
+                <div class="tech-title">
+                    <img src="./images/circle.png" alt="">
+                    <h5>Skills</h5>
+                </div>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+           
+            
+        </div>
+        <ul id="about-my-self">
+            <li class="tool-category">
+              <h4 class="heading1">Language</h4>
+              <ul class="list-heading">
+                <li class="single-heading">Java Script</li>
+                <li class="single-heading">Ruby</li>
+                <li class="single-heading">Html</li>
+                <li class="single-heading">Css</li>
+              </ul>
+            </li>
+            <li class="tool-category">
+              <h4 class="heading2">Framworkes</h4>
+              <ul class="list-heading">
+                <li class="single-heading">Ruby on Rails</li>
+                <li class="single-heading">Bootstrap</li>
+                <li class="single-heading">Selenium</li>
+                <li class="single-heading">RSPec</li>
+                <li class="single-heading">CapyBara</li>
+              </ul>
+            </li>
+            <li class="tool-category">
+              <h4 class="heading3">Skills</h4>
+              <ul class="list-heading">
+                <li class="single-heading">GitHub</li>
+                <li class="single-heading">Codekit</li>
+                <li class="single-heading">Codepen</li>
+                <li class="single-heading">Gitlab</li>
+                <li class="single-heading">Ubuntu</li>
+              </ul>
+            </li>
+          </ul>
+    </section>
+    
+    <section id="contact-sec" class="contact-me">
+        <div class="contact-main">
+            <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+            <form action="https://formspree.io/f/myyazqal" method="post">
+                <div class="input-group">
+                    <div class="name-text">
+                        <input type="text" name="name" id="name" placeholder="Full Name">
+                        <hr>
+                    </div>
+                </div>
+                <input type="email" name="email" id="email" placeholder="Email address">
+                <hr class="form-seprator">
+                <textarea name="message" id="message" placeholder="Hello Oscar. I would like to get in touch with you"></textarea>
+                <button type="submit">Get in touch</button>
+            </form>
+        </div>
```

Comment:
- [ ] You have done excellently well with the media query issues but your contact section got messed up in the desktop view after resolving previous issues and I am sure you would like to make this look good to future recruiters or viewers. It is not looking so good at the moment. Please consider the images below and see how you can resolve these `height`, `width`, and `display` issues:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/215186816-be201d60-939f-4b03-af97-35b61c4dd0db.png)|![image](https://user-images.githubusercontent.com/99715640/215186910-bf482fe3-2daa-4d03-b783-5f66a4732162.png)|